src/components/layout/layout.tsx

```diff
@@ -123,9 +131,11 @@ const Layout: React.FC<LayoutProps> = (props) => {
         siteTitle={data.site.siteMetadata.title}
         siteTitleUrl={props.siteTitleUrl}
         $lightTheme={isLight}
-        heroImage={props.heroImage}
```

Comment:
I moved the hero from inside he header into the layout. Makes it much easier to understand and to extend.